data/phi0631/phi001/phi0631.phi001.perseus-eng2.xml

```diff
@@ -207,11 +207,11 @@ So Silius Italicus, xv. 84:
 <p>By these two virtues, intrepidity in war, and equity in peace, they maintained themselves and their state. Of their exercise of which virtues, I consider these as the greatest proofs; that, in war, punishment was oftener inflicted on those who attacked an enemy contrary to orders, and who, when commanded to retreat, retired too slowly from the contest, than on those who had dared to desert their standards, or, when pressed by the enemy,<note anchored="true" place="foot">IX. Pressed by the enemy] <quote xml:lang="lat">Pulsi.</quote> In the words <foreign xml:lang="lat">pulsi loco cedere ausi erant, loco</foreign> is to be joined, as Dietsch observes, with <foreign xml:lang="lat">cedere,</foreign> not, as Kritzius puts it, with <foreign xml:lang="lat">pulsi.</foreign> "To retreat," adds Dietsch, " is disgraceful only to those <foreign xml:lang="lat">qui ab hostibus se pelli patiantur,</foreign> who suffer themselves to be repulsed by the enemy."</note> to abandon their posts; and that, in peace, they governed more by conferring benefits than by exciting terror, and, when they received an injury, chose rather to pardon than to revenge it.</p></div>
 <div type="textpart" subtype="chapter" n="10">
 <p> But when, by perseverance and integrity, the republic had increased its power; when mighty princes had been vanquished in war;<note anchored="true" place="foot">X. When mighty princes had been vanquished in war] Perses, Antiochus, Mithridates, Tigranes, and others.</note> when barbarous tribes and populous states had been reduced to subjection; when <placeName key="tgn,7016143">Carthage</placeName>, the rival of <placeName key="tgn,7013962">Rome</placeName>'s dominion, had been utterly destroyed, and sea and land lay <pb n="15"/>every where open to her sway, Fortune then began to exercise her tyranny, and to introduce universal innovation. To those who had easily endured toils, dangers, and doubtful and difficult circumstances, ease and wealth, the objects of desire to others, became a burden and a trouble. At first the love of money, and then that of power, began to prevail, and these became, as it were, the sources of every evil. For avarice subverted honesty, integrity, and other honorable principles, and, in their stead, inculcated pride, inhumanity, contempt of religion, and general venality. Ambition prompted many to become deceitful; to keep one thing concealed in the breast, and another ready on the tongue;<note anchored="true" place="foot">To keep one thing concealed in the breast, and another ready on the tongue] <quote xml:lang="lat">Aliud clausum in pectore, aliud in linguâ promptum, habere.</quote>
-<quote><foreign xml:lang="grc">Ἐχθρὸσ γάρ μοι κεῖνοσ ὁμῶσ Ἀΐδαο πύλῃσιν</foreign><lb/>
-<foreign xml:lang="grc">Ὃσ χ</foreign><lb/>
+<quote><foreign xml:lang="grc"><l>Ἐχθρὸσ γάρ μοι κεῖνοσ ὁμῶσ Ἀΐδαο πύλῃσιν</foreign></l>
```

Comment:
There is an error here because the close `</l>` is outside of the `<foreign>` tag.
Overall it would probably be simpler to make the whole quote "grc" as opposed to these two foreign tags anyhow. (This is something I would do when cleaning up).

Reply:
I have fixed this in the latest comment, thanks as always for your watchful eye :)